hugo/content/research/2019/_index.md

```diff
@@ -3,11 +3,14 @@ title: "DORA Research: 2019"
 date: 2019-10-01
 draft: false
 research_year: 2019
-# SEM and survey questions are included via template: templates/research_archives/single.html, if specified in front matter. The data for survey questions can be found at data/survey_questions.json
-sections_to_display:
-    - 'Structural Equation Model'
 type: research_archives
+layout: single
+tab_order: "0"
+tab_title: "Overview"
 ---
 
-# DORA's Research Program: 2019
-DORA's research program continued in 2019. [Read the 2019 report here]({{< relref "dora-report" >}})
+![Infographic p1](infographic/2019-DORA-Report-Infographic-1.svg)
+![Infographic p2](infographic/2019-DORA-Report-Infographic-2.svg)
```

Comment:
The bottom of the infographic has big words that say "DOWNLOAD THE FULL REPORT" and has a non-clickable URL.

Suggest linking (at least) the second page of the infographic to the download the report page.

`[![Infographic p2](infographic/2019-DORA-Report-Infographic-2.svg)](/research/2019/dora-report/)`

or 
`[![Infographic p2](infographic/2019-DORA-Report-Infographic-2.svg)](https://doradotdev-staging--pr432-lirjkjf9.web.app/research/2019/dora-report/2019-dora-accelerate-state-of-devops-report.pdf)`

Reply:
I've been running into struggles today getting links to work in SVGs (not helped by trying to adjust to a new version of Illustrator!) so I'm going to move this to a new issue and decouple it from the rest of this one.